crates/nu-path/Cargo.toml

```diff
@@ -13,9 +13,11 @@ bench = false
 
 [dependencies]
 dirs = { workspace = true }
+cfg-if = "1.0.0"
```

Comment:
Is it possible to remove this dependency?

Reply:
Removed OK